### PR TITLE
Add GeoStyler Code Sprint 2023 blog post

### PR DESCRIPTION
**Pull request by jansule**
This adds the GeoStyler Code Sprint 2023 blog post

geostyler-code-sprint-2023-blog-post

### CODE REVIEW

File: content/codesprint-2023/contributions.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+During the Code Sprint, we achieved the following:
+
+<img src="/images/geostyler-codesprint-2023-stats.png" alt="GeoStyler Code Sprint Stats" height="800px" />
+
+<br />
+
+### GeoStyler CLI
+
+The [GeoStyler CLI](https://github.com/geostyler/geostyler-cli) can now additionally be used as standalone executable! There is no need to install NodeJs anymore, if you just want to convert between different styling formats without using the UI. The executables can be downloaded for each version from the [repository](https://github.com/geostyler/geostyler-cli/releases). We support Linux, MacOS and Windows.
```
Comment (dnlkoch) [inline content/codesprint-2023/contributions.md:19]:
```suggestion
The [GeoStyler CLI](https://github.com/geostyler/geostyler-cli) can now additionally be used as standalone executable! There is no need to install Node.js anymore, if you just want to convert between different styling formats without using the UI. The executables can be downloaded for each version from the [repository](https://github.com/geostyler/geostyler-cli/releases). We support Linux, MacOS and Windows.
```

**Review by dnlkoch — APPROVED** [review]

File: content/codesprint-2023/contributions.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+![GeoStyler UI Modal Example](/images/geostyler-codesprint-2023-modals.gif)
+
+Kudos to [@dnlkoch](https://github.com/dnlkoch), [@SchulA](https://github.com/SchulA), [@jansule](https://github.com/jansule) and [@hwbllmnn](https://github.com/hwbllmnn).
+
+### GeoStyler Mapbox Parser
+
+We spent quite some time discussing the [geostyler-mapbox-parser](https://github.com/geostyler/geostyler-mapbox-parser) and how we can integrate the concept of map-based styles (which mapbox is using) in addition to layer-based styles (which SLD is using) in GeoStyler. We came to the conclusion that layers in mapbox fit quite well to the idea of rules in layer-based styles. Both are entities consisting of filters, scale denominators and styling instructions. With the support for expressions in GeoStyler in particular, we should now be able to advance the development of the geostyler-mapbox-parser and make it a first class citizen in the GeoStyler ecosystem.
+
+Besides that, the geostyler-mapbox-parser is now able create lossless conversions between mapbox styles and geostyler-styles.
```
Comment (KaiVolland) [inline content/codesprint-2023/contributions.md:57]:
```suggestion
```

Thats not true. :)

**Review by KaiVolland — COMMENTED** [review]
Very nice. I think you could add some prose between Sponsors and Contributions. somehting like:
"One of the strength of the GeoStyler is it's wide range of possibilities. So was the wide range of different usecases for the contributors. This results in many cool contributions to all parts of the GeoStyler ecosystem." (no spell checks or grammar checks :stuck_out_tongue: )

Also a small conclusion in the post would be nice. Just one or two sentences.

**Review by marcjansen — APPROVED** [review]
Very nice, maybe you can add the changes wrt the opencollective so people know that they can easiyl sponsor us now. You decide.

File: content/codesprint-2023/contributions.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jansule) [reply content/codesprint-2023/contributions.md]: Rephrased

Comment (jansule) [reply content/codesprint-2023/contributions.md]: Rephrased

**Pull request merged**